Examples/Detectors/TelescopeDetector/include/ActsExamples/TelescopeDetector/TelescopeDetector.hpp

```diff
@@ -0,0 +1,36 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/Detector/IBaseDetector.hpp"
+#include "ActsExamples/Utilities/OptionsFwd.hpp"
+
+#include <memory>
+#include <vector>
+
+namespace ActsExamples {
+namespace Telescope {
+class TelescopeDetectorElement;
+}  // namespace Telescope
+}  // namespace ActsExamples
+
+struct TelescopeDetector : public ActsExamples::IBaseDetector {
```

Comment:
Is there a reason that `TelescopeDetector` is not in the `ActsExamples` namespace?

Reply:
Not from my side, but I see that all other detectors are not included any namespace.